content/en/docs/tasks/configure-pod-container/quality-service-pod.md

```diff
@@ -42,7 +42,7 @@ kubectl create namespace qos-example
 
 ## Create a Pod that gets assigned a QoS class of Guaranteed
 
-For a Pod to be given a QoS class of Guaranteed:
+For a Pod to be given a QoS class of `Guaranteed`:
 
```

Comment:
Nit, but lowercase container throughout this doc would be nice to avoid it sounding like an API object